Tools/Audit-Sentinel-Detection/Readme.md

```diff
@@ -0,0 +1,80 @@
+
+
+# Audit Sentinel Detection Rules
+
+### Prerequistes : [Configure Audit in Microsoft Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/enable-monitoring)
+
+### Purpose : Detection / Analytics are dircetly changed in the production instace at times due to several reaons.
```

Comment:
@samikroy, please fix the below typos 

dircetly to directly 
instace to instance
 reaons to reasons

Reply:
Fixed